fklub-vedtaegter.tex

```diff
@@ -138,7 +138,7 @@ \section{Bestyrelsen og de frecolor-keephue: trueaste udvalg}
 \item Bestyrelsen fordeler selv sine arbejdsopgaver, der som minimum
   består af at holde kontakt med de autonome F-klub-udvalg, at
   være ansvarlig for faglige arrangementer, og at holde styr med
-  F-klubbens IT- og papirsvirke.
```

Comment:
Tænker det er fint at tage den på dansk, når det er vedtægter og ikke kode;
Jeg læser 'papirsvirke' som F-Klubbens regnskab m.m., altså at bestyrelsen sørger for at udvalgene fører regnskab, og andet papir relateret.

Antager der er et 's' for at binde det samme med det forrige ord "IT-"